packages/redux-routine/src/is-action.js

```diff
@@ -3,12 +3,13 @@
  */
 import { isPlainObject, isString } from 'lodash';
 
+/* eslint-disable jsdoc/valid-types */
 /**
  * Returns true if the given object quacks like an action.
  *
- * @param {*} object Object to test
+ * @param {any} object Object to test
```

Comment:
Is there a reason for using `any` here instead of `unknown`?

Reply:
Yes because we access a property on it we actually need `any`.

Reply:
Because it could be anything, not just a Record, which represents a specific mapping between two types. 🙂

Reply:
Got it! Learning every day :)